app/helpers/layout_helper.rb

```diff
@@ -329,20 +330,6 @@ def form_to_submit_id(f)
     "aid_#{key}_#{model}"
   end
 
-  def submit_or_cancel(f, overwrite = false, args = { })
```

Comment:
The file was too long and rubocop generated a too long error so i moved it to application.